dashboard/README.md

```diff
@@ -32,11 +32,24 @@ Start a RisingWave cluster, create some tables and materialized views for testin
 For example:
 
 ```bash
-./risedev d
+./risedev d full
 ./risedev slt  e2e_test/nexmark/create_sources.slt.part
 ./risedev psql -c 'CREATE TABLE dimension (v1 int);'
 ./risedev psql -c 'CREATE MATERIALIZED VIEW mv AS SELECT auction.* FROM dimension join auction on auction.id-auction.id = dimension.v1;'
-./risedev psql -c 'INSERT INTO dimension select 0 from generate_series(1, 50);'
+./risedev psql -c 'CREATE MATERIALIZED VIEW mv2 AS SELECT * FROM mv;'
+./risedev psql -c 'CREATE MATERIALIZED VIEW mv3 AS SELECT count(*) FROM mv2;'
+
+./risedev psql -c 'CREATE MATERIALIZED VIEW mv4 AS SELECT * FROM mv;'
+./risedev psql -c 'CREATE MATERIALIZED VIEW mv5 AS SELECT count(*) FROM mv2;'
+./risedev psql -c 'CREATE MATERIALIZED VIEW mv6 AS SELECT mv4.* FROM mv4 join mv2 using(id);'
+./risedev psql -c 'CREATE MATERIALIZED VIEW mv7 AS SELECT max(id) FROM mv;'
+
+./risedev psql -c 'CREATE MATERIALIZED VIEW mv8 AS SELECT mv.* FROM mv join mv6 using(id);'
+./risedev psql -c 'CREATE SCHEMA s1;'
+./risedev psql -c 'CREATE TABLE s1.t1 (v1 int);'  
+./risedev psql -c 'CREATE MATERIALIZED VIEW s1.mv1 AS SELECT s1.t1.* FROM s1.t1 join mv on s1.t1.v1 = mv.id;'
+
+./risedev psql -c 'INSERT INTO dimension select 0 from generate_series(1, 20);'
```

Comment:
This looks like this:
<img width="936" alt="Screenshot 2024-09-19 at 4 13 27 PM" src="https://github.com/user-attachments/assets/b7f1a571-4450-440a-b37f-5b028e1d7e82">